extensions/positron-python/python_files/posit/positron/positron_ipkernel.py

```diff
@@ -480,6 +482,26 @@ def __init__(self, **kwargs) -> None:
         # Patch haystack-ai to ensure is_in_jupyter() returns True in Positron
         patch_haystack_is_in_jupyter()
 
+    # -----
+    # Backport https://github.com/ipython/ipykernel/pull/1210 which removes the control queue and
```

Comment:
would it be helpful to log something to check back in if this gets solved upstream?